recipes/thorvg/all/conanfile.py

```diff
@@ -129,6 +129,9 @@ def requirements(self):
         if self.settings.os == "Linux":
             if self.options.with_engines in ["gl", "gl_beta"]:
                 self.requires("opengl/system")
+        if self.options.with_threads and Version(self.version) >= "0.15.0":
+            if self.options.get_safe("with_openmp") and self.settings.compiler in ["clang", "apple-clang"]:
```

Comment:
@toge `with_openmp` option does not exist either in Conanfile or upstream.

Reply:
As far as I saw, it only depends on the `threads` one: https://github.com/thorvg/thorvg/blob/b4238cd647f0d54fd4536c26cdf9d8ac99e0c747/src/renderer/sw_engine/meson.build#L23